File: Java/lab22/src/main/java/org/example/Consultant.java
package org.example;

public class Consultant extends WorkerTemplate{
    @Override
    void doWork() {
        System.out.println("I help clients");
    }

    @Override
    void takeOver() {
        System.out.println("I have a break from 13:00 to 14:00");
    }

    @Override
    void continueWork() {
        System.out.println("I continue help clients");
    }
}

File: Python/lab22/Consultant.py
from WorkerTemplate import WorkerTemplate


class Consultant(WorkerTemplate):
    def do_work(self):
        print("I help clients")

    def take_over(self):
        print("I have a break from 13:00 to 14:00")

    def continue_work(self):
        print("I continue helping clients")